components/articles/individual-article/comments/CommentAdder.jsx

```diff
@@ -6,21 +6,33 @@ function CommentAdder(props){
     const {setComments, articleID} = props
     const {signedInUser} = useContext(UserContext)
     const [commentTextbox, setCommentTextbox] = useState("")
+    const [error, setError] = useState("")
     function handleSubmit(event){
         event.preventDefault()
-        setCommentTextbox("")
         postComment(articleID, signedInUser, commentTextbox).then((postedComment) => {
             setComments((currentCommentsList) => {
                 const commentsListCopy = [...currentCommentsList]
                 commentsListCopy.unshift(postedComment)
                 return commentsListCopy
             })
+            setCommentTextbox("")
+        }).catch((err) => {
+            if(err.response.data.message === "Comment body must not be empty"){
+                return setError(err.response.data.message)
+            }
+            setError("Your comment could not be posted. Please try again later.")
         })
     }
-    return (<form>
+    return (<form onSubmit={handleSubmit}>
         <label for="comment-textbox">Add a comment here:</label>
-        <input id="comment-textbox" type="text" onChange={(event) => {setCommentTextbox(event.target.value)}}></input>
-        <input type="submit" onClick={handleSubmit}></input>
+        <input 
```

Comment:
I'm already working on this change! The text area works fine as of now, but I'm wondering how/if it would be possible to have the height of the text area change automatically once the user fills up the text area so that it adds a bit of extra height for the user.